dashboard/client/src/components/ActorTable.tsx

```diff
@@ -448,6 +452,12 @@ const ActorTable = ({
                         ip={address?.ipAddress}
                         type=""
                       />
+                      <br />
+                      <MemoryProfilingButton
+                        pid={pid}
+                        ip={address?.ipAddress}
+                        type=""
```

Comment:
Should we just not pass in type instead of passing in an empty string? 

Same in the test case